recipes/hhsuite/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+
+mkdir build
+cd build
+cmake -DHAVE_MPI=0 -DCMAKE_INSTALL_PREFIX="${PREFIX}" -DHAVE_SSE2=1 ..
+make -j 2
+make install
```

Comment:
is there something like a `make test` command as well?

Reply:
Unfortunately, we do not have any tests for hhsuite. So there is no `make test` command.

Reply:
Ok, let's see if this works now.

Reply:
Thanks a lot!